billingClient/billingClient.go

```diff
@@ -0,0 +1,42 @@
+package billingclient
+
+import (
+	"context"
+
+	e3dbClients "github.com/tozny/e3db-clients-go"
+	"github.com/tozny/e3db-clients-go/authClient"
+)
+
+const (
+	BillingServiceBasePath = "v1/billing"
+	InternalCreateCustomer = "/internal/create_customer"
```

Comment:
Internal paths should follow the current pattern of `internal/apiVersion/serviceName/path`.

Reply:
👍 will update